parsing/parsetree.mli

```diff
@@ -220,9 +217,9 @@ and pattern_desc =
         (* `A             (None)
            `A P           (Some P)
          *)
-  | Ppat_record of (Longident.t loc * pattern) list * closed_flag
-        (* { l1=P1; ...; ln=Pn }     (flag = Closed)
-           { l1=P1; ...; ln=Pn; _}   (flag = Open)
+  | Ppat_record of (Longident.t loc * pattern option) list * closed_flag
+        (* { l1; ...; ln=Pn }        [(l1, None); ...; (ln, Some Pn)], Closed
+           { l1=P1; ...; ln=Pn; _}   [(l1, None); ...; (ln, Some Pn)], Open
```

Comment:
I think you want to drop the second `P1` as well

Reply:
Yes, thanks.